static/dbview.config.php

```diff
@@ -664,6 +664,7 @@
 			"question-end-time" => ["post-question", "end-time"],
 			"has-categories" => "0",
 			"has-media" => "EXISTS(SELECT `id` FROM `post-media` WHERE `post-media`.`uri-id` = `post-thread`.`uri-id`)",
+			"total-comments" => "(SELECT COUNT(*) FROM `post` WHERE `parent-uri-id` = `post-thread`.`uri-id` AND `gravity` = 6)",
```

Comment:
> Statuses that have been interacted with more than others.

Doesn't this include like and reshares? Also there should be a time window for this interaction count, like over the past 3 days, in order to cycle the trends regularly.

Reply:
The time window is more complicated to achieve concerning the performance. And filtering likes and reshares is not easily possible on this level. We mustn't simply count all activities since there are some technical ones as well that aren't visible.

Reply:
Yes but in the present state this endpoint will always show the posts that had the most comments of all time, which isn't likely to change much. I'm not sure we should support this endpoint if we can't provide meaningful data.

Reply:
No. The query in the endpoint only fetches posts that had been commented on in the last 24 hours.

Reply:
Oh, okay, that's good then.

Reply:
Also I added a second condition so that only posts are considered that had been commented on in the last 24 hours and also had been created during the last week. This avoids some really long running posts. For example on Diaspora there is a monster thread that started several years ago and comments still flushing in. But the current query is solving this.